app/kudos/views.py

```diff
@@ -128,17 +128,6 @@ def marketplace(request):
     if q:
         title = f'{q.title()} Kudos'
         token_list = token_list.keyword(q)
-        # log this search, it might be useful for matching purposes down the line
-        try:
-            SearchHistory.objects.update_or_create(
-                search_type='kudos',
-                user=request.user,
-                data=request.GET,
-                ip_address=get_ip(request)
```

Comment:
@x5engine what do you think of just removing the ip_address portion?

Reply:
yes why not but the whole app has many ip addresses